### DISCUSSION TITLE
GitHub Copilot could not connect to server. Extension activation failed: "read ECONNRESET"

### DISCUSSION THREAD

**Discussion by GitHub-Cool** [community/community#38366] [Copilot Conversations] [answered]
Copilot Output
`[ERROR] [default] [2022-11-05T17:13:47.075Z] GitHub Copilot could not connect to server. Extension activation failed: "read ECONNRESET"
`
I have programmed with the Copilot for a long time.
Yesterday, I found that Copilot couldn't work.
I tried to modify the hosts file and DNS,It still doesn't work.
In WebStorm, when I enter the right code, 'Waiting for GitHub Authentication...' and no next step.
I'm not sure if there's a problem with my network,but I can access copilot-proxy.githubusercontent.com|api.github.com with v2ray.
Copilot is really  an amazing project. Please give me a solution.plz.

**Comment by Guf8844** [comment]
> Выход второго пилота `[ERROR] [default] [2022-11-05T17:13:47.075Z] GitHub Copilot could not connect to server. Extension activation failed: "read ECONNRESET" ` Я программировал с помощью второго пилота в течение длительного времени. Вчера я обнаружил, что второй пилот не может работать. Я пытался изменить файл hosts и DNS, это все равно не работает. В WebStorm, когда я ввожу правильный код, «Ожидание аутентификации GitHub…» и никакого следующего шага. Я не уверен, есть ли проблема с моей сетью, но я могу получить доступ к copilot-proxy.githubusercontent.com|api.github.com с v2ray. Copilot — действительно потрясающий проект. Пожалуйста, дайте мне решение.plz.

Очем эт вы тут  ше##читесь ать🤓😇😉💰

**Reply by GitHub-Cool** [reply]
hello,I don't understand what you are saying

**Comment by Guf8844** [comment]
- [ ] 66778:_&&&"привет))

**Comment by GitHub-Cool** [comment, marked as answer]
vscode proxy setting

**Reply by muhBilal** [reply]
how to solved this, please tell me...

**Comment by ArnoZWang** [comment]
My solution was restarting everything over again, including:
- Restart Copilot
- Reinstall Copilot
- Restart VS Code
- Restart proxy tool if you are using any
- Reboot computer

I cannot guarantee which step was working, but it worked.
